### ISSUE TITLE
customize dataset

### ISSUE THREAD

**Issue by ITPreson** [MVIG-SJTU/AlphAction#22]
Hello, I have a few questions.

1. I'd like to know why we don't use 'person_bbox' during training

2. I don't know what "# disable box_file when train, use only gt to train" means. Isn't box the ground truth?

3. I want to customize the dataset. My action data usually lasts about 3 seconds with varying length, but I see the clip of AVA as one second, I first came into contact with the concept of keyframes, and why should I take the first frame as a keyframe, which can represent the action classification of this clip? I am very confused. Can you give me the general idea?

looking forward to you reply.

**Comment by yelantf** [comment]
For question 1 and 2, we use ground truth bounding boxes with jittering for training and we do not use bounding boxes detected by other human detectors (like faster RCNN). We also tried using detected bounding boxes(i.e. box_file) for training before, but the result is worse than using jittered ground-truth bounding boxes. Using both ground-truth boxes and detected boxes could perform even better, but we did not experiment with this setting.

For question 3, in AVA, short segments (±1.5 seconds centered on a keyframe) are used to provide temporal context for labeling the actions in the middle frame. The human bound boxes are annotated only on the keyframe to reduce the annotation cost, while the action categories are determined by the content of the segments(whether the segment contains a given action).
